### ISSUE TITLE
Get '301: moved permanently' on some envs

### ISSUE THREAD

**Issue by mcascone** [janjoerke/vscode-jenkins-pipeline-linter-connector#21]
On my Linux environments, I get a `301: moved permanently` error when running the linter. 

```html
<html>
<head><title>301 Moved Permanently</title></head>
<body bgcolor="white">
<center><h1>301 Moved Permanently</h1></center>
<hr><center>nginx/1.6.3</center>
</body>
</html>
```

What could cause this? I have all the same settings as windows servers that work correctly.

**Comment by Greg3482** [comment]
Have you checked if your Url settings in extensions are using the correct http:// or https:// this caught me out and gives a `301: moved permanently`.
